ballerina/Module.md

```diff
@@ -28,7 +28,7 @@ message HelloResponse {
 gRPC allows client applications to directly call the server-side methods using the auto-generated stubs. Protocol
  Buffer compiler is used to generate the stubs for the specified language. In Ballerina, the stubs are generated using the built-in 'Protocol Buffers to Ballerina' tool. 
 
-For information on how to generate Ballerina code for Protocol Buffers definition, see [gRPC](https://ballerina.io/learn/user-guide/network-communication/grpc/).
+For information on how to generate Ballerina code for Protocol Buffers definition, see [the gRPC user guide](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

Comment:
```suggestion
For information on how to generate Ballerina code for Protocol Buffers definition, see [Write a gRPC service with Ballerina](https://ballerina.io/learn/write-a-grpc-service-with-ballerina/).
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/761/commits/60af9102b24b4960998e0b9ca322e6a7602ae741